README.md

```diff
@@ -81,11 +81,12 @@ async fn main() -> Result<(), Box<dyn Error>> {
     tokio::spawn(handle_request(tracer.clone()));
 
     // Start to report.
-    let handle = tracer.reporting(async move {
-        let _ = signal::ctrl_c().await;
-    });
-
-    handle.await?;
+    tracer
+        .reporting()
+        .with_graceful_shutdown(async move {
+            let _ = signal::ctrl_c().await;
+        })
+        .await?;
```

Comment:
We need to add sync mode in the doc, and provide a clear explanation about when should use sync, and what would be the possible impact? Such as, sync mode usually means it would block the current thread until reported.